README.md

```diff
@@ -42,6 +42,8 @@ Join the slack! <https://join.slack.com/t/findthemasks/shared_invite/zt-czdjjznp
 ?hide-list={true/false} (also hides filters)
 ?hide-search={true/false} (beta)
 ?locale={see available locales in dropdown at https://findthemasks.com}
+?q={map search term: '100 Fake Road, Kansas, MO'}
+?coords={lat,lng}&zoom=12 // zoom only works in concert with coords
```

Comment:
@ngottlieb just curious would normal people just Google a lat/lng they want via "what's here?" and put it in there?

Reply:
by normal people do you mean like, you and other outreach volunteers, or a hospital administrator or something? For the former, I think yes. My quickest way to find latlng of a location is to open google maps and then just click on a spot on the map and latlng gets added both to the browser URL bar and a little label at the bottom of the page.

Reply:
haha yes non-dev wizards. Okay, so same manual process - got it!